Projects/mnemos

```diff
@@ -0,0 +1,26 @@
+# mnemos
+
+## Name
```

Comment:
We can probably remove the "name" field, I think we only need it once for the title :)